Deployment Guides/Provisioning-Islandora-on-Ubuntu-16.md

```diff
@@ -0,0 +1,1796 @@
+
+sed -i "s|cache.server.worker.enabled = false|cache.server.worker.enabled = true|" cantaloupe.properties
+
+sed -i "s|resolver.static = FilesystemResolver|resolver.static = HttpResolver|" cantaloupe.properties
+
+sed -i "s|HttpResolver.trust_all_certs = false|HttpResolver.trust_all_certs = true|" cantaloupe.properties
+
+# - DigiBESS and Islandora Vagrant differ on these
+#sed -i "s|endpoint.iiif.content_disposition = none|endpoint.iiif.content_disposition = inline|" cantaloupe.properties
+
+#sed -i "s|processor.limit_to_8_bits = true|processor.limit_to_8_bits = false|" cantaloupe.properties
+
+#sed -i "s|cache.server.derivative.enabled = false|cache.server.derivative.enabled = true|" cantaloupe.properties
+
+#sed -i "s|cache.server.ttl_seconds = 2592000|cache.server.ttl_seconds = 259200|" cantaloupe.properties
+
+#sed -i "s|FilesystemCache.pathname = /var/cache/cantaloupe|FilesystemCache.pathname = /srv/cantaloupe/cache|" cantaloupe.properties
+
+#sed -i "s|HttpResolver.lookup_strategy = BasicLookupStrategy|HttpResolver.lookup_startegy = ScriptLookupStrategy|" cantaloupe.properties
+
+#sed -i "s|HttpResolver.BasicLookupStrategy.url_prefix = http://localhost/images/|HttpResolver.BasicLookupStrategy.url_prefix =|" cantaloupe.properties
+
+#sed -i "s|HttpResolver.auth.basic.username =|HttpResolver.auth.basic.username = anonymous|" cantaloupe.properties
+
+#sed -i "s|HttpResolver.auth.basic.secret =|HttpResolver.auth.basic.secret = anonymous|" cantaloupe.properties
+```
+
+Islandora Vagrant uses a Cantaloupe provided script, called `delegates-3.4.rb` that permits "obfuscation" of URLs. We leave further configuration of this service as an exercise to the reader. See the DigiBESS site.
+
+
+#### Setup Logging <a id="setup-logging"></a>
+Please note logging still needs some TLC: log4j and logrotate clash with some files: 
+``` 
+cd ~ 
+
+git clone https://github.com/discoverygarden/islandora_log_config.git
+
+cd islandora_log_config
+
+cp log4j.xml $CATALINA_HOME/webapps/fedoragsearch/WEB-INF/classes/log4j.xml 
+
+cp -p $CATALINA_HOME/conf/logging.properties $CATALINA_HOME/conf/logging.properties.bak
+
+cp logging.properties $CATALINA_HOME/conf/logging.properties
+
+### Chose one of these, depending on where you installed Adore-Djatoka, with Fedora or with Blazegraph
+# - with Fedora
+#cp log4j.properties $CATALINA_HOME/webapps/adore-djatoka/WEB-INF/classes/log4j.properties
+# - with Blazegraph
+#cp log4j.properties /usr/share/tomcat7-blzg/webapps/adore-djatoka/WEB-INF/classes/log4j.properties
+# - Also do the following if you installed Adore-Djatoka with Blazegraph
+#sed -i "s|/usr/local/fedora/server/logs|/var/bigdata/logs|" /usr/share/tomcat7-blzg/webapps/adore-djatoka/WEB-INF/classes/log4j.properties
+
+cp logback.xml $FEDORA_HOME/server/config/logback.xml
+
+chown -R blazegraph:blazegraph /usr/share/tomcat7-blzg/
+
+systemctl restart blazegraph
+
+cd ~
+
+rm -rf islandora_log_config
+```
+
+#### Drupal Filter <a id="drupal-filter"></a>
```

Comment:
Should checkout 
https://github.com/discoverygarden/fcrepo3-security-jaas/blob/master/README.md
https://github.com/discoverygarden/fcrepo3-security-jaas/blob/master/docs/multisite-optimization.md

Reply:
Updated the drupal filter to 7.1.11, however the 'fcrepo3-security-jass'
library wouldn't compile, I think due to a dependency on 7.1.10.

Reply:
Updated. Tested on a machine and it works for me.